app/assets/styles/customization/espoo/customization.scss

```diff
@@ -111,11 +111,16 @@
     padding-top: 50px;
   }
 
-  .app-HomePage .app-HomePageContent__banner
+  .app-HomePage
     div {
       background-color: $blue-light;
```

Comment:
This selector looks a bit scary to me, could we make it more precise than: "all divs under .app-HomePage"?

Reply:
agree, but i don't even know if it have some side-effect if we target only 1 div in our fix..

Reply:
Ah nvm, i guess it's just selector, will remove without any hurts